cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
         return data
```

Comment:
This validation method looks a bit odd. You're calling a class method on the `Ticket` model, but passing in a `ValidationError` class. This could potentially lead to confusion. It might be better to raise the `ValidationError` directly in the serializer's `validate` method.